src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -22,9 +22,10 @@
   private final RequestQueue requestQueue;
   private final ServicePoint checkInServicePoint;
   private final Request highestPriorityFulfillableRequest;
+  private UUID loggedInUserId;
```

Comment:
Is this field intended to be mutable? Why would the value change after initialisation?

Reply:
done.